docs/source/README.rst

```diff
@@ -0,0 +1,39 @@
+.. _README:
```

Comment:
they are not SymLinks.  Would you prefer them to be SymLinks?  This might sort out some issues with Read the Docs not functioning correctly.